data-otservbr-global/npc/asima.lua

```diff
@@ -41,6 +49,12 @@ local itemsTable = {
 		{ itemName = "ultimate spirit potion", clientId = 23374, buy = 438 },
 		{ itemName = "vial", clientId = 2874, sell = 5 },
 	},
+	["creature products"] = {
+		{ itemName = "spellwand", clientId = 651, sell = 299 },
+	},
+	["spellbook"] = {
```

Comment:
It should be shields category, if I'm not wrong spellbooks are handle as shields.